composer.json

```
@@ -15,5 +15,20 @@
     "platform": {
       "php": "7.4"
     }
+  },
+  "autoload": {
+    "psr-4": {
+      "Jeedom\\plugins\\": "plugins/"
```

```suggestion
      "Jeedom\\Plugins\\": "plugins/"
```

pour respecter la PSR-4

---

Non je ne force rien du tout cas j'ai laissé l'autoloader spécifique existant pour les plugins, celui dans core.inc.php.

ça permet juste aux futurs plugins d'utiliser l'autoloader normalisé PSR-4 de Composer au lieu de celui spécifique à Jeedom. Mais ça ne permet pas de remplacer la classe de base du plugin qui doit rester avec la convention Jeedom et sans namespace `[pluginid].class.php` ni même la classe `*cmd.class.php`